### DISCUSSION TITLE
'no symbolic links' option

### DISCUSSION THREAD

**Discussion by tjcw** [bfgroup/b2#424] [Q&A]
For reasons of getting a Windows build of another project to work, I want to build BOOST with copies of the header files in the boost/ directory rather than symbolic links. I have raised a feature issue here https://github.com/bfgroup/b2/issues/423 .
I know how to get `b2` to do the copies; the patch needed is this
```
--- a/src/tools/link.jam
+++ b/src/tools/link.jam
@@ -36,6 +36,8 @@ actions touch {
 
 rule can-symlink ( project )
 {
+    .can-symlink = false ;
+    return false ;
     if ! $(.can-symlink)
     {
         local root-project = [ get-root-project $(project) ] ;
```
but I don't know how to do this conditionally; ideally it would depend on a command-line option such as `./b2 nosymlinks` . Can anyone tell me how to do this ?

**Comment by tjcw** [comment]
It looks like what I want can be done with
```boost install --prefix ...
```

**Reply by gabrieldevillers** [reply]
@tjcw it looks like some info was lost in your reply.

**Reply by tjcw** [reply]
I figured out how to do it with "b2 install"
